test/blocks/global-footer/test-utilities.js

```diff
@@ -0,0 +1,93 @@
+import { setViewport } from '@web/test-runner-commands';
+import { setConfig } from '../../../libs/utils/utils.js';
+import { config, loadStyles, viewports } from '../global-navigation/test-utilities.js';
+import { waitForElement } from '../../helpers/waitfor.js';
+
+export const containerSelector = '.global-footer';
+
+export const allSelectors = {
```

Comment:
The best solution I found, to avoid refactoring too much, is to import the values from the gnav utilities in the footer utilities and use them. In this way, if `.feds-navLink` now needs to be `.feds-nav-link`, as in your example, there's just one place that this change needs to be done.

`import { selectors as keyboardSelectors } from '../../../libs/blocks/global-navigation/utilities/keyboard/utils.js';

export const allSelectors = {
  container: keyboardSelectors.globalFooter,
  footerIcons: '.feds-footer-icons',
.......`

Reply:
I think this is a good approach overall, as we don't have to duplicate selectors 👍